SU-HKKU/cardinal_service_profile
# CSD-202 Style fixes

# READY FOR REVIEW

# Summary
- Style fixes

# Needed By (Date)
- 8/24

# Urgency
- medium

# Steps to Test

1. Pull this branch
2. Clear site cache
3. Review the following (will update shortly)
- Dashboard Opportunity page button should be styled (for empty and list state)
- User page and dashboard Opportunity page should have top and bottom margin.
- Opportunity row: Location should span the length of When and Open To combined (see [CSD-336](https://stanfordits.atlassian.net/browse/CSD-336)) 
- Opportunity row: Spacing between Application deadline and Location shouldn't be equal. Application deadline height should flex based on the content (see [CSD-342](https://stanfordits.atlassian.net/browse/CSD-342)) 
- Homepage one-column overlay should now have margin-bottom (dependent on https://github.com/SU-SWS/jumpstart_ui/pull/69) 
- Opportunity, Resource, and Story titles for list items should all be `font-weight: 700`
- Search button should be centered for all list pages when shifting to its own line (at 992 breakpoint). 

Additional User-link styles and tweaks will be included in a separate fix-up ticket.

# Associated Issues and/or People
- [CSD-202](https://stanfordits.atlassian.net/browse/CSD-202)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
